src/App.jsx

```diff
@@ -1,13 +1,10 @@
 import "./App.css"
-import Header from "./components/client/common/header/Header.jsx"
-import HeroSection from "./components/client/landing/heroSection/HeroSection.jsx"
-
+import Home from "./pages/Home/home"
```

Comment:
Maintain index file inside Home folder and import as below.
import Home from "./pages/Home"

Reply:
Done